src/content/concepts/mode.md

```diff
@@ -4,6 +4,10 @@ sort: 4
 contributors:
   - EugeneHlushko
   - byzyk
+  - mrichmond
+related:
+  - title: 'webpack options defaulter source code'
```

Comment:
`options defaulter` is the name of the class which is not covered here. Perhaps a bit more generic title would fit better here, something like: "webpack default options (source code)" or something

Reply:
makes sense 👍